### PR TITLE
Added `PagerDuty` service on keptn artifacthub

### PR DESCRIPTION
**Pull request by afzal442**
This PR supports `PagerDuty` integration with `keptn` webhook
Closes: https://github.com/keptn/integrations/issues/18

cc @agardnerIT

### CODE REVIEW

**Comment by afzal442** [comment]
cc @grabnerandi @oleg-nenashev @robertjahn @christian-kreuzberger-dtx

File: pagerduty/1.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# PagerDuty Integration
+
+[PagerDuty](https://support.pagerduty.com/docs/) is a project management and incident monitoring platform. This page describes how to integrate Keptn with PagerDuty.
+
+![pd model](https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/pagerduty/1.0.0/assets/pd2.png)
+
+## Overview
+TODO
```
Comment (oleg-nenashev) [inline pagerduty/1.0.0/README.md:8]:
```suggestion

```

File: pagerduty/1.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The [keptn webhook service](https://github.com/keptn/keptn/tree/master/webhook-service), which comes "out of the box" with Keptn, will be used to listen for Keptn events and trigger webhooks into PagerDuty. In this way new PagerDuty projects, tasks or anything else in PagerDuty are created in response to a Keptn cloudevent.
```
Comment (StackScribe) [inline pagerduty/1.0.0/README.md:9]:
```suggestion
The [keptn webhook service](https://github.com/keptn/keptn/tree/master/webhook-service) and documented on the [Webhook Integration](https://keptn.sh/docs/0.19.x/integrations/webhooks/) page, which comes "out of the box" with Keptn, will be used to listen for Keptn events and trigger webhooks into PagerDuty. In this way new PagerDuty projects, tasks or anything else in PagerDuty are created in response to a [Keptn CloudEvent](https://keptn.sh/docs/0.19.x/reference/miscellaneous/events/)
```

File: pagerduty/1.0.0/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - Create a new task every time a load test or evaluation is finished. Include the results and any output of the test / evaluation
+  - Tag PagerDuty users in tasks to notify them that action is required
+
+## Step 1: Gather PagerDuty Details
+TODO
+
+## Step 2: Keptn Webhook Setup
+> This can be done programmatically or via the Keptn's bridge
+
+1. Navigate to a keptn project and to go the integrations screen
```
Comment (StackScribe) [inline pagerduty/1.0.0/README.md:22]:
```suggestion
1. Navigate to a Keptn project and to go the integrations screen
```

File: pagerduty/1.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  "routing_key": "55ddf4914b4a4904c0776c7896eab557",
+  "dedup_key": "srv01/qa",
+  "event_action": "trigger",
+  "client": "My Datadog Service",
+  "client_url": "http://www.datadog.co/service/observability"
+}
+```
+
+That's it! Execute the sequence and an PagerDuty service task/incidentS will be automatically created.
```
Comment (StackScribe) [inline pagerduty/1.0.0/README.md:48]:
```suggestion
That's it! Execute the sequence and PagerDuty service task/incidentS will be automatically created.
```

**Comment by afzal442** [comment]
It'll be great if it can be reviewed before KCD. 😄

**Review by agardnerIT — CHANGES_REQUESTED** [review]
`The keptn webhook service and documented` > `The keptn webhook service is documented`
`In this way new PagerDuty projects` > `In this way, new PagerDuty projects`
`response to a Keptn CloudEvent` > `response to a Keptn CloudEvent.`
`Navigate to our Sign Up page,...` > `[Sign up](https://www.pagerduty.com/sign-up/) for a PagerDuty account`

`This can be done programmatically` how? You don't tell me. Please tell me or (better) point me to docs that tell me.

`service task/incidentS` > `service tasks / incidents`

Please add `webhook` and `notification` as keywords to `artifacthub-pkg.yml`

**Comment by afzal442** [comment]
Thanks much @agardnerIT for your review and suggestions here. That really reminded me of @StackScribe who scooped up for any typos and all.

**Comment by afzal442** [comment]
Thanks @agardnerIT for heads up! I made it now. Have a look again.

**Comment by afzal442** [comment]
Heyya folks, can I get reviews on it again? I hope we can make it before this year ends, 😉 

cc @grabnerandi

**Review by agardnerIT — APPROVED** [review]
LGTM. Let's merge and make any new changes moving forwards. Sorry for the delay!

**Pull request merged**